multiversx/mx-chain-go
# Fix missing log event.

## Reasoning behind the pull request
- Fixed the problem when the log event with `CompletedTxEvent` identifier is missing in case of a smart contract call that transfer value cross-shard.
  
## Proposed changes
- Changed the condition when the log event with the identifier `CompletedTxEvent` is generated for a value transfer through a smart contract result.

## Testing procedure
- Deploy on a testnet a smart contract that transfers EGLD value to a given address. Check that the smart contract result with the value transfer has a log with the identifier `CompletedTxEvent`.

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
